zallet/build.rs

```diff
@@ -352,3 +356,74 @@ pub(super) static METHODS: ::phf::Map<&str, RpcMethod> = ::phf::phf_map! {
 
     Ok(())
 }
+
+fn generate_debian_copyright(target_dir: &Path) -> Result<(), Box<dyn Error>> {
+    let mut contents = "Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: zallet
+
+Files:
+ *
+Copyright: 2024-2025, The Electric Coin Company
+License: MIT OR Apache-2.0"
+        .to_string();
+
+    let licensing = embed_licensing::collect(embed_licensing::CollectConfig::default())?;
```

Comment:
The generation does slow down builds a bit, I suspect due to this live-fetching everything each time something changes that re-runs the build script. But we can resolve that with targeted caching later (i.e. ensuring that this only runs if dependencies change).

Reply:
Actually, we only need this in release builds, so I'll disable it for debug builds.